arq/jobs.py

```diff
@@ -66,6 +66,15 @@ def __init__(
         self._queue_name = _queue_name
         self._deserializer = _deserializer
 
+    async def abort(
+        self, timeout: Optional[float] = None, *, pole_delay: float = 0.5, key_expire: Optional[int] = 10
+    ) -> Any:
```

Comment:
I'm not clear on the return type here - what does it mean? I think it should be `bool`